package.json

```diff
@@ -30,7 +30,8 @@
   "files": [
     "dist/background-process.js",
     "dist/collect.js",
-    "dist/spawn-background-process.js"
+    "dist/spawn-background-process.js",
+    "scripts/postinstall.js"
   ],
   "scripts": {
```

Comment:
Do we need to define a postinstall script here?

Reply:
removed and added `dist/notify.js`, I do think we need it so that it's published?